exercises/accumulate/babel.config.js

```diff
@@ -0,0 +1,13 @@
+const presets = [
+  [
+    '@babel/env',
+    {
+      targets: {
+        node: 'current',
+      },
+      useBuiltIns: false,
+    },
+  ],
+];
+
+module.exports = { presets };
```

Comment:
Sorry if I am missing something, why do we need to separate babel config from package.json?

Reply:
Because we shouldn't store it in package.json just like we shouldn't store eslint rules in package.json. Either move both out or neither.

Reply:
https://stedolan.github.io/jq/